src/assembler.ts

```diff
@@ -220,6 +224,7 @@ export class Assembler implements Emitter {
       jsiiVersion,
       bin: this.projectInfo.bin,
       fingerprint: '<TBD>',
+      usedFeatures: this.usedFeatures.size > 0 ? Array.from(this.usedFeatures) : undefined,
```

Comment:
This probably comes from my branch, but I wonder if we should stick a `.sort()` on here to guarantee test stability.

```suggestion
      usedFeatures: this.usedFeatures.size > 0 ? Array.from(this.usedFeatures).sort() : undefined,
```